_posts/2020-10-13-bigsur-new.md

```diff
@@ -0,0 +1,329 @@
+
+The goal is to combine OpenCore into Clover allowing the project to live a bit longer, as Clover's current state can no longer boot macOS Big Sur or older versions of OS X such as 10.6. As of writing, this project seems to be a bit confusing as there seems to be little reason to actually support Clover. Many of Clover's properties have [feature-parity in OpenCore](https://github.com/dortania/OpenCore-Install-Guide/tree/master/clover-conversion) and trying to combine both C++ and C ruins many of the features and benefits either languages provide. The only feature OpenCore does not support is macOS-only ACPI injection, however the reasoning is covered here: [Does OpenCore always inject SMBIOS and ACPI data into other OSes?](https://dortania.github.io/OpenCore-Install-Guide/why-oc.html#does-opencore-always-inject-smbios-and-acpi-data-into-other-oses)
+
+## Death of x86 and the future of Hackintoshing
+
+With macOS Big Sur, a big turning point is about to happen with Apple and their Macs. As we know it, Apple will be shifting to in-house designed Apple Silicon Macs(Really just ARM) and thus x86 machines will slowly be phased out of their lineup within 2 years. 
+
+What does this mean for both x86 based Macs and Hackintoshing in general? Well we can expect about 5 years of proper OS support for the iMac20,x series which released earlier this year with an extra 2 years of security updates. After this, we Apple will most likely stop shipping x86 builds of macOS and hackintoshing as we know it will have passed away.
+
+For those still in denial and hope something like ARM Hackintoshes will arrive, please consider the following:
+
+* We have yet to see an iPhone "Hackintosh" and thus the likely hood of an ARM Hackintosh is unlikely as well
+* Apple's use of "Apple silicon" hints that ARM is not actually what future macs will be running, instead we'll see highly customized chips *based* off ARM
+  * For example, Apple will be heavily relying on hardware features such as [W^X](https://en.wikipedia.org/wiki/W%5EX), kernel memory protection, Pointer Auth, etc for security and thus both macOS and Applications will be dependant on it. This means hackintoshing on bare-metal(without a VM) will become extremely difficult without copious amounts of work
+  * Also keep in mind Apple Silicon will no longer be UEFI-based like Intel Macs are, meaning a huge amount of work would also be required on this end as well
+
+So while we may be heart broken the journey is coming to a stop in the somewhat near future, hackintoshing will still be a time piece in Apple's history. So enjoy it now while we still can, and we here at Dortania will still continuing supporting the community with our guides till the very end!
+
+# Getting ready for macOS 11, Big Sur
+
+This will be your short run down if you skipped the above:
+
+* DRM is broken for many
+  * Specifically for those relying on WhateverGreen for DRM
+* Many Ivy Bridge and Haswell SMBIOS were dropped
```

Comment:
```suggestion
* Some kexts are broken (like MacProMemoryNotificationDisabler) because Lilu user-patching has severely broke.
* Many Ivy Bridge and Haswell SMBIOS were dropped
```